github/pom.xml

```diff
@@ -72,4 +82,22 @@
     </dependency>
 
   </dependencies>
+
+  <build>
+
+    <plugins>
+
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <configuration>
+          <ignoredUnusedDeclaredDependencies>
+            <ignoredUnusedDeclaredDependency>org.keycloak:keycloak-spring-security-adapter</ignoredUnusedDeclaredDependency>
```

Comment:
perhaps can be removed, marks that it's ok to have it declared even if it's not used directly